launch/start-node/start-node.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+// A very simple binary, tasked with starting the processes for both
+// the blind bid and the node.
+func main() {
+	blindBidCmd := exec.Command("./blindbid")
+	if err := blindBidCmd.Start(); err != nil {
+		fmt.Printf("could not start blind bid binary - %v\n", err)
+		os.Exit(1)
+	}
+
+	nodeCmd := exec.Command("./testnet")
+	if err := nodeCmd.Start(); err != nil {
+		fmt.Printf("could not start node binary - %v\n", err)
```

Comment:
In case of errors the program should return with an error code (like the above)

Reply:
Ah yeah, good catch. Totally forgot it there